src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -58,16 +59,30 @@ def get_data(
         data_keys.POPULATION.TMRLE: load_theoretical_minimum_risk_life_expectancy,
         data_keys.POPULATION.ACMR: load_forecasted_mortality,
         data_keys.POPULATION.LIVE_BIRTH_RATE: load_standard_data,
-        data_keys.ALZHEIMERS.PREVALENCE_SCALE_FACTOR: load_alzheimers_prevalence,
+        data_keys.POPULATION.SCALING_FACTOR: load_alzheimers_all_states_prevalence,
         data_keys.ALZHEIMERS.PREVALENCE: load_alzheimers_prevalence,
-        data_keys.ALZHEIMERS.INCIDENCE_RATE: load_standard_data,
+        data_keys.ALZHEIMERS.BBBM_CONDITIONAL_PREVALANCE: load_bbbm_conditional_prevalence,
+        data_keys.ALZHEIMERS.MCI_CONDITIONAL_PREVALENCE: load_mci_conditional_prevalence,
+        data_keys.ALZHEIMERS.INCIDENCE_RATE: load_standard_data,  # GBD cause incidence
+        data_keys.ALZHEIMERS.STATE_INCIDENCE_RATE: load_ad_dementia_state_incidence_rate,  # state incidence for transition from MCI
+        data_keys.ALZHEIMERS.BBBM_INCIDENCE_COUNT: load_bbbm_incidence_count,
+        # MCI incidence rate caluclated during sim using mci_hazard.py and time in state
         data_keys.ALZHEIMERS.CSMR: load_standard_data,
         data_keys.ALZHEIMERS.EMR: load_standard_data,
         data_keys.ALZHEIMERS.DISABLIITY_WEIGHT: load_standard_data,
+        data_keys.ALZHEIMERS.MCI_DISABILITY_WEIGHT: load_mci_disability_weight,
         data_keys.ALZHEIMERS.RESTRICTIONS: load_metadata,
         data_keys.ALZHEIMERS.TOTAL_POPULATION_INCIDENCE_RATE: load_alzheimers_total_population_incidence,
     }
-    return mapping[lookup_key](lookup_key, location, years)
+    mapped_value = mapping[lookup_key](lookup_key, location, years)
+    # drop age_groups under 5
```

Comment:
```suggestion

    # To avoid issues with irrelevant differences between the very young age
    # groups in GBD and FHS data, we will drop rows with age_start < 5 from
    # all age-specific DataFrames
```